### ISSUE TITLE
OpenDV/DummyRepeater/DV3000/ Missing Makefile

### ISSUE THREAD

**Issue by mwbesemer** [dl5di/OpenDV#166]
The Makefile for OpenDV/DummyRepeater/DV3000/ is missing.

**Comment by dl5di** [comment]
No. The Makefile is created automatically from Makefile.am by Automake.
Use "./configure" to build it.
